docs/src/docs/asciidoc/user/clustered-cache.adoc

```diff
@@ -151,14 +151,14 @@ On Windows:
 [source,cmd]
 ----
 cd <path/to/terracotta/kit>/server/bin
-start-tc-server.bat -f <path/to/server/config>/tc-config.xml
+start-tc-server.bat --config-file clustered/client/src/test/resources/configs/docs/cluster.properties
```

Comment:
This was just an example of how to use the startup script with a config file. You don't need to specify an actual path here. Saying `start-tc-server.bat -f path/to/config-file` suffices.

I'd collapse the unix example into this one and simply use `start-tc-server.(bat|sh) -f path/to/config-file`.

Also, it would be good idea to mention the usage of the script without the config file.